Common/src/Storage/ResultTableExtensions.cs

```diff
@@ -255,4 +255,22 @@ public static async Task<IEnumerable<ResultIdStatus>> GetResultStatus(this IResu
                                     cancellationToken)
                         .ToListAsync(cancellationToken)
                         .ConfigureAwait(false);
+
+  /// <summary>
+  ///   Set the result status to <see cref="ResultStatus.DeletedData" />
+  /// </summary>
+  /// <param name="resultTable">Interface to manage results</param>
+  /// <param name="resultId">Id of the result to update</param>
+  /// <param name="cancellationToken">Token used to cancel the execution of the method</param>
+  /// <returns>
+  ///   Task representing the asynchronous execution of the method
+  /// </returns>
+  public static async Task MarkAsDeleted(this IResultTable resultTable,
```

Comment:
I am not sure it is needed to create an extension method for that as it is used only once in the code.

Reply:
That was done like that for the others